packages/wrangler/src/pages/dev.ts

```diff
@@ -537,9 +543,9 @@ export const Handler = async ({
 		}),
 		persist,
 		persistTo,
-		inspect: undefined,
+		inspect,
 		logPrefix: "pages",
-		logLevel: logLevel ?? "warn",
+		logLevel: logLevel ?? "info",
```

Comment:
Arguably this is a (very minor) breaking change.
There might be people who are parsing the outgoing logs from `wrangler pages` and not expect to see so many messages.
It might be safer not to include this change?

Reply:
I think we should aim to do the same as `wrangler dev` proper, which's default is `log`.

Reply:
Considering the possible impact on Users, we would still want to semver it as a breaking change. There should also be more planned work on the messaging API of Wrangler Core (I think Pete has mentioned customizability before) which can be part of the efforts that go towards v3 much like the Programmatic API

Reply:
Nah, this isn't a change that would justify waiting until wrangler v3. `pages` commands still have a beta warning, and it's purely a logging/stdout change which we've never guaranteed anyway.

Reply:
I think you can get away with this due to the beta label on pages commands. But apart from that we need to start being careful changing things that we didn't expect to be public surface. Once stuff is out there we lose control over how it is relied upon.